README.md

```diff
@@ -22,9 +22,10 @@ For MoveIt 2.0 development, see [moveit2](https://github.com/ros-planning/moveit
 
 service    | Indigo | Kinetic | Melodic | Master
```

Comment:
Can we drop Indigo? It's not built anymore!
Try running [`create_readme_table.py`](https://github.com/ros-planning/moveit/blob/967f6f667acba2dba18b51e9a516555365776ed2/moveit/scripts/create_readme_table.py). It should generate the table, but maybe it needs an update...

Reply:
I wasn't able to get  this to work for me locally... when I run this I get a bunch of 404 errors.

Reply:
I added a commit to the end to drop Indigo from the first table.  It seems that that file is just for generating the second table and I can't get it to work for me locally.  I'm not sure what I'm doing wrong or if something needs to be updated in that script.

Reply:
I didn't update the python file that generates these but I did go and manually remove indigo from the readme.